test/functional/rpc_psbt.py

```diff
@@ -195,5 +197,49 @@ def run_test(self):
         # Test decoding error: invalid base64
         assert_raises_rpc_error(-22, "TX decode failed invalid base64", self.nodes[0].decodepsbt, ";definitely not base64;")
 
+        # Send to all types of addresses
+        addr1 = self.nodes[1].getnewaddress()
+        txid1 = self.nodes[0].sendtoaddress(addr1, 11)
+        vout1 = find_output(self.nodes[0], txid1, 11)
+        addr2 = self.nodes[1].getnewaddress()
+        txid2 = self.nodes[0].sendtoaddress(addr2, 11)
+        vout2 = find_output(self.nodes[0], txid2, 11)
+        addr3 = self.nodes[1].getnewaddress()
+        txid3 = self.nodes[0].sendtoaddress(addr3, 11)
+        vout3 = find_output(self.nodes[0], txid3, 11)
+        self.sync_all()
+
+        # Update a PSBT with UTXOs from the node
+        # Inputs should not be filled because they are witness
```

Comment:
13932: inputs should  _not_ be filled because they are _non-_ witness

Reply:
Resolved in latest push! Good catch!